src/sign/index.ts

```diff
@@ -1,6 +1,6 @@
 import { createHmac } from "crypto";
 
-export function sign(secret: string, payload: string | object): string {
+export function sign(secret?: string, payload?: string | object): string {
```

Comment:
can you revert this change? Both these arguments really are required, and we don't want to suggest otherwise.